src/metabase/public_settings.clj

```diff
@@ -113,6 +113,20 @@
   :visibility :public
   :audit      :getter)
 
+
+  (defsetting help-link
+  (deferred-tru "TODO")
```

Comment:
I have no idea what those things do besides allowing us to send the new settings, we'll need to get some help from BE